packages/wallets/src/routes/Router.tsx

```diff
@@ -58,13 +72,43 @@ const Router: React.FC = () => {
     const { data: walletAccounts, isLoading } = useWalletAccountsList();
 
     if ((!walletAccounts || !walletAccounts.length) && !isLoading)
-        return <Route component={WalletNoWalletFoundState} path={walletsPrefix} />;
+        return (
+            <Route
+                path={walletsPrefix}
+                render={() => (
+                    <React.Suspense fallback={null}>
+                        <LazyWalletsNotFoundState />
+                    </React.Suspense>
+                )}
+            />
+        );
 
     return (
         <Switch>
-            <Route component={CompareAccountsRoute} path={`${walletsPrefix}/compare-accounts`} />
-            <Route component={CashierModalRoute} path={`${walletsPrefix}/cashier`} />
-            <Route component={WalletsListingRoute} path={walletsPrefix} />
+            <Route
+                path={`${walletsPrefix}/compare-accounts`}
+                render={() => (
+                    <React.Suspense fallback={null}>
```

Comment:
minor/optional: is there a need for loader as fallback?

Reply:
The components themselves have skeleton loaders included, but I can add a simple loader here just in case